yolopass-backend/app/controllers/admin/events_controller.rb

```diff
@@ -0,0 +1,21 @@
+class Admin::EventsController < ApplicationController
+  before_action :authenticate_user
+  before_action :authorize_admin
```

Comment:
@PranaliM0 Why is it still present in each controller, please check previous comments again ?